frontends/election-manager/src/utils/generate_batch_tally_results_csv.test.ts

```diff
@@ -57,7 +57,7 @@ describe('generateBatchTallyResultsCSV', () => {
       'Batch ID,Batch Name,Tabulator,Number of Ballots'
     );
     expect(headerRow).toContain(
-      '"Liberty Party Governor - Ballots Cast","Liberty Party Governor - Undervotes","Liberty Party Governor - Overvotes","Liberty Party Governor - Aaron Aligator","Liberty Party Governor - Peter Pigeon","Liberty Party Governor - Write In",'
+      '"Liberty Party Governor - Ballots Cast","Liberty Party Governor - Undervotes","Liberty Party Governor - Overvotes","Liberty Party Governor - Aaron Alligator","Liberty Party Governor - Peter Pigeon","Liberty Party Governor - Write In",'
```

Comment:
The competing candidate, the Pigeon [🕊️](https://www.mentalfloss.com/article/554182/what-is-difference-between-pigeons-and-doves) isn't misspelled, so I see no particular reason to misspell the Alligator 🐊 (yes, the emoji is for a Crocodile, but there isn't an Alligator emoji, and see other comment...).